appium/tests/specs/inbox/CheckReplyAndForwardForEncryptedEmail.spec.ts

```diff
@@ -19,12 +19,13 @@ describe('INBOX: ', () => {
 
     const replySubject = `Re: ${emailSubject}`;
     const forwardSubject = `Fwd: ${emailSubject}`;
-    const quoteText = `On 11/5/21 at 4:15 AM ${senderEmail} wrote:\n > ${emailText}`;
+    const quoteText = `On 11/5/21 at 1:15 PM ${senderEmail} wrote:\n > ${emailText}`;
```

Comment:
I keep seeing people switch this line this back end forth between these two results. It seems to depend on the device that the test runs on, and its timezone, or some similar environment problem we cannot control.

Use a regex instead, to allow the time to be any number.